index.php

```diff
@@ -0,0 +1,5 @@
+<?php
+include('f1trklstarray.php');
```

Comment:
Names of files, variables, etc should be as clear as possible. This one it's too hard to read. Try something like `track_list.php`